beetsplug/absubmit.py

```diff
@@ -164,8 +164,13 @@ def _get_analysis(self, item):
                     item=item, error=e
                 )
                 return None
+            decoded = bytearray()
             with open(filename, 'r') as tmp_file:
-                analysis = json.load(tmp_file)
+                byte = tmp_file.read(1)
+                while byte:
+                    decoded.extend(bytes(byte, 'utf-8'))
+                    byte = tmp_file.read(1)
```

Comment:
Then we don't need to try to encode the string byte by byte. We can just read the binary data directly: for example, with `f.read()`.